Server/Main.cs

```diff
@@ -523,7 +541,7 @@ public static void Main(string[] args)
 
 				Console.WriteLine(" - Press return to exit, or R to try again.");
 
-				if (Console.ReadKey(true).Key != ConsoleKey.R)
+				if (Insensitive.Equals(Console.ReadLine(), "R"))

```

Comment:
This is not working.  Insensitive.Equals(Console.ReadLine(), "R") is not reading the first r on the line.  You either have to put a capital R or you have to put more than one "r".